tests/suites/test_suite_x509parse.function

```diff
@@ -1472,3 +1472,83 @@ exit:
     ;;
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_X509_CRT_PARSE_C */
+void x509_crt_parse_subjectkeyid(data_t *buf, unsigned int subjectKeyIdLength, int ref_ret)
```

Comment:
Sorry, you can't use `unsigned int` as the type of a test function argument. Until yesterday, the test framework was sloppy and accepted `unsigned int` but treated it as `int` under the hood. Since #6716 you can use larger _signed_ integer types (up to `intmax_t`), but unsigned types are rejected. A follow-up will make unsigned types work, but it's not ready yet.

Reply:
Modified test function parameters to `int`.